PROJECT.md

```diff
@@ -44,8 +44,8 @@ See the [`LICENSE`](./LICENSE) file for details.
 
 ```
 /**
- * Copyright (C) <year(s)> Data Together
- * This program is free software: you can redistribute it and/or modify it
+ * Copyright (C) <year(s)> Data Together.
```

Comment:
And then this one as well

Reply:
I want to make the case for the trailing period one more time and then will accept whatever you want to choose here.

I submit to you that the text the FSF has in their templates was never meant to lack a period at the end of the Copyright line. I believe they left it out simply because they have a placeholder at the end of the Copyright line. As evidence for this, I offer the following examples of archetypical GNU software by the FSF:

* Random file from GNU `make`: https://github.com/mirror/make/blob/master/main.c
* Random file from `gcc`: https://github.com/gcc-mirror/gcc/blob/master/libtool.m4
* Random file from `ls`: https://github.com/coreutils/coreutils/blob/master/src/chmod.c

All of them have a period at the end of that line.  So either they're not following their own guidelines in their own software, or it is acceptable to put a period at the end of the Copyright line.